src/args.rs

```diff
@@ -86,6 +85,6 @@ pub fn parse_config(path: PathBuf) -> Config {
         eprintln!("Could not parse config file: {}", e);
         ::std::process::exit(2)
     });
-    println!("{:?}", config);
+    debug!("{:?}", config);
```

Comment:
If you want to do this, you need to move the initialization of the stderrlog before config parsing.

Reply:
done in additional commit.